src/logoIcons.ts

```diff
@@ -0,0 +1,13 @@
+import horizontal from './logoIcons/horizontal.svg';
+import horizontalInvert from './logoIcons/horizontalInvert.svg';
+import vertical from './logoIcons/vertical.svg';
+import verticalInvert from './logoIcons/verticalInvert.svg';
```

Comment:
These are just internals and we can change these later on if really needed; we could rename the svg-files to tell that these are Suomifi logos as we could have also other logos in this new category.

Reply:
Good point. I guess it'll make sense if/when we end up with some other logos as well.